modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,78 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+====
+
+In the following table replace _version_ with the actual product version string.
+
+[cols="1,1", options="header"]
+.Hardware and Software Requirements
+|===
+| Requirement            | Description
+| openSUSE Leap 15.1     | Clean installation, up-to-date
+| Multi-core CPU         | x86_64
+| 8 GB RAM *) 	         | Sufficient for testing purposes
+| 100 GB Disk Space **)  | Disk space depends on your channel requirements
+|===
+
+*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).
+
+**) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.
```

Comment:
I'm not sure what the close parentheses are doing here. You might also like to consider replacing the asterisks with proper icons. We use `icon:asterisk[role="none"]` elsewhere in the suite.

Reply:
Ok, I will go for `icon:asterisk[role="none"]`.  (Without something directly after the *, adoc treats it as list item. Maybe, I coud have escaped them with backslashes...)